en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -230,6 +230,65 @@ When there are null values in custom expressions and are:
 * Fields without attributes, the null value is ignored. If the value in a required field, an error message occurs. If the value is in an optional field, the value is ignored and not sent to the connector. +
 * Fields with attributes, the value is ignored and not sent to the connector.
 
+=== Create a Picklist Mapping
+
+Using Picklist Mapping, you can map data from one application to the corresponding data needed by another application and then reuse those mappings in different flows in your organization. This allows you to connect two applications that have different values for similar fields and to align the values in those fields across applications. This enables you to determine how the possible values of an output field (data pill) map to the possible values of an input field.
+
+For example, you can map a data pill from the *Country* field in a Salesforce record and provide the same values in the *Country* field of the NetSuite record.
+
+[%header%autowidth.spread]
+|===
+|Country (Salesforce) | Country (NetSuite)
+|US |_unitedStates
+|CA |_canada
+|DE |_germany
+|AR |_argentina
+|===
+
+In this example, the *Status* field in a Case in Salesforce is mapped to the *Status* field in a Ticket in Zendesk to create continuity in field values across systems.
+
+[%header%autowidth.spread]
+|===
+|Status (Salesforce) | Status (NetSuite)
+|New |New
+|Working |Open
+|Waiting on Customer |On-Hold
+|Escalated |Open
+|Closed |Closed
+|===
+
+When mapping source data fields:
+
+* The source field must be a data pill with a fixed list of values.
+* The target field must be an enumerated xref:ms_composer_custom_expression_editor.adoc#data-types[data type] field.
+* The default mapping specifies the value in the target system if any other unmapped value from the source system is found. If you do not specify a default value, then the original value from the source field is passed to the target field.
+* The mapping is stored at the organization level and can be reused across flows.
+* The mapping must have a unique name.
+* Mappings are available to everyone in your organization.
+
+To map source data fields to target fields:
+
+. From the *Composer Flows* tab, open a flow.
+. Click *Edit*.
+. In an action card, in a field that supports *Pick from List*, select *Picklist Mapping* from the drop-down menu. +
+The *Picklist Mapping* window appears.
+. In the *Picklist Source* section, in the *Source* field, select the value from the drop-down list that you want to be mapped to the value in the *Target* field, and then click *Next*.
+. In the *Global Picklist Selection* section:
```

Comment:
. To create a new picklist, in the *Global Picklist Selection* section, click *Create new picklist*.
. In the *Global picklist name* field, enter a unique name for the picklist.
. Optionally, in the *Description* field, enter a description for the picklist. The following description is populated automatically based on your source and target systems: +
`This maps [Source Field Name] from [Source Connection Name] to [Target Field Name] from [Target Connection Name]`
. In the table, select the source fields and their target mappings.
. Optionally, to add additional field mappings, click *Add Row* and enter the source field values manually (with a row representing each value) and select the target mappings from the drop-down.
. After you add all applicable field mappings, click *Create Picklist*.
. Click *Apply*. +

Reply:
I was trying to edit this to get rid of all the nested steps because I generally dislike them. I don't know if this works or not, but I'd at least try to keep it at 2 levels

Reply:
One thing you could maybe do is break these into two optional tasks

=== Create a new picklist
=== Choose an existing picklist